src/components/BigButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+interface BigButtonProps {
+  color?: string;
+  onChange?: () => void;
+}
+
+const BigButton = styled.button<BigButtonProps>`
+  background-color: ${props => props.color || 'blue'};
+  color: white;
+  padding: 10px 20px;
```

Comment:
Ideally it's good practice to use `rem` units instead of `px`! By default, most browsers have it set such that `1rem` = `16px`, so we can do unit conversion to adjust that. The reason why is so things scale nicely with `font-size`.

Reply:
https://nekocalc.com/px-to-rem-converter